tor_ocr/main.py

```diff
@@ -53,11 +53,12 @@ def process_image(image_url):
     json_result = decode_image_from_url(image_url)
 
     result = {
-        'text': json_result['ParsedText'],
+        'text': json_result['ParsedResults'][0]['ParsedText'],
```

Comment:
This will still explode on an error state because the ParsedText field won't be there. We might want to do some kind of check on the code returned first to see if we can route it through this parser or make a special error state parser instead.